### DISCUSSION TITLE
mmyolo更换主干网络后，为什么要保持neck、head的widefacotr=1.0

### DISCUSSION THREAD

**Discussion by 2022jay** [open-mmlab/mmyolo#490] [Q&A]
尝试替换其他系数，例如yolov5s的0.5，但是结果报错！如果不能更改系数，那么网络neck层的参数量会特别大，该如何解决呢？

**Comment by RangeKing** [comment]
out_channels 由 widen_factor 控制，`[最终的 out_channels] = [配置文件里的 out_channels] * widen_factor`。设置 `widefactor = 1.0` 只是为了方便，只要 backbone、neck 以及 head 的 int_channels 和 out_channels 能够对应上就行了。您可以检查一下自定义的配置文件里是不是都对应好了。
